{{cookiecutter.project_name}}/.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -23,67 +23,22 @@ body:
   - type: textarea
     id: versions
     attributes:
-      label: Version information
+      label: Versions
       description: |
-        Please paste below the output of
+        Which version of packages.
+
+        Please install `session-info2`, run the following command in a notebook,
+        click the “Copy as Markdown” button, then paste the results into the text box below.
 
         ```python
-        import session_info
-        session_info.show(html=False, dependencies=True)
+        In[1]: import session_info2; session_info2.session_info(dependencies=True)
```

Comment:
@flying-sheep do we need to import the package here before we run the rest? Yes, right? While possible, it'd be a bit annoying to get the cookiecutter variable in here.

Reply:
With `dependencies=True` not necessarily, but you should paste what it does into the `placeholder` anyway instead of removing it, so people can see how it’s supposed to look like

Reply:
This is adapted from anndata:
https://raw.githubusercontent.com/scverse/anndata/refs/heads/main/.github/ISSUE_TEMPLATE/bug-report.yml

Still want to keep the placeholder paste?

Reply:
I think it probably helps, what do you think?